modules/sample/src/main/resources/petstore.json

```diff
@@ -788,8 +788,7 @@
         "description": "",
         "operationId": "loginUser",
         "produces": [
-          "application/xml",
```

Comment:
This left because we didn't know what `application/xml` was? Any reason why this can't just be handled by handling the "best" one for now? My concern is folk who don't have direct ownership of the swagger doc to the point where they mandate an incomplete spec be curated just for them

Reply:
I needed something to test text/plain with and I guess hadn't noticed contentType-textPlain.yaml yet.  Will revert this part.